### ISSUE TITLE
Disabling "Use animations" has unintended side effects on other packages

### ISSUE THREAD

**Issue by nikolowry** [atom-material/atom-material-ui#271]
This was first reported in issue https://github.com/silvestreh/atom-material-ui/issues/131- however, we were unable to collectively pin point why certain users could/couldn't reproduce the bugs.

After setting up a new machine and doing a fresh install - I was able to deduce that the "Use animations" being disabled was the culprit. I'm assuming other packages might have dependent events and event listeners on their UI elements that get overridden and thus "breaking" their packages.

The offending style:
```css
atom-workspace:not(.use-animations) *, 
atom-workspace:not(.use-animations) *::before, 
atom-workspace:not(.use-animations) *::after{
    transition:none !important;
    -webkit-animation-duration:0s !important;
}
```

I'd suggest maybe targeting elements(buttons, inputs, etc) more specifically, rather then simply disabling all transitions/animations on a global scope.

**Comment by James-Byrne** [comment]
I can confirm that turning on animations fixed #225 for me

**Comment by silvestreh** [comment]
Thanks @James-Byrne. Now I have another reason to fix this one 😀
